AccessibilityInsightsForAndroidService/.idea/runConfigurations.xml

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
do you remember why this file was removed? we might want to `.gitignore` this file as well if it was deleted as part of debugging something else.